app/src/pages/model_results.py

```diff
@@ -771,48 +808,105 @@ def model_results():
+                            parameter="Stage",
+                            start_date=start_date,
+                            end_date=end_date,
+                            data_type="iv",
+                            reference_df=ref_line_wse_ts,
+                        )
+                        if gage_stage_ts.empty:
+                            gage_stage_ts = pd.DataFrame(columns=["time", "obs_wse"])
+                        else:
+                            gage_stage_ts["obs_wse"] = (
+                                gage_stage_ts["obs_stage"] + gage_datum
+                            )
+                        # Get the Flow Data
                         obs_flow_ts = query_s3_obs_flow(
                             st.session_state["s3_conn"],
                             st.session_state["pilot"],
                             feature_gage_id,
                             st.session_state["gage_event"],
                         )
-                        obs_flow_ts.rename(columns={"flow": "obs_flow"}, inplace=True)
-                        gage_flow_ts = obs_flow_ts.merge(
-                            ref_line_flow_ts, on="time", how="outer"
-                        )
-                        info_col.markdown("### Observed vs Modeled Flow")
-                        with info_col.expander(
-                            "Time Series Plots",
-                            expanded=False,
-                            icon="📈",
-                        ):
-                            st.write("")
-                            plot_ts(
-                                obs_flow_ts,
-                                ref_line_flow_ts,
-                                "obs_flow",
-                                "model_flow",
-                                dual_y_axis=False,
-                                title=feature_label,
+                        if obs_flow_ts.empty:
+                            # try getting instantaneous values
+                            gage_flow_ts = query_nwis(
+                                site=feature_gage_id,
+                                parameter="Streamflow",
+                                start_date=start_date,
+                                end_date=end_date,
+                                data_type="iv",
+                                reference_df=ref_line_flow_ts,
                             )
-                        with info_col.expander("Data Table", expanded=False, icon="🔢"):
+                        else:
+                            gage_flow_ts = obs_flow_ts.merge(
+                                ref_line_flow_ts, on="time", how="outer"
+                            )
+                        info_col.markdown("### Observed vs Modeled Flow")
+                    with info_col.expander(
+                        "Time Series Plots",
+                        expanded=True,
+                        icon="📈",
+                    ):
+                        if not gage_flow_ts.empty:
```

Comment:
Getting this now with `mill-creek`/`mill-creek_to_chambers-creek`/`breach_may2015`:
```
UnboundLocalError: cannot access local variable 'gage_flow_ts' where it is not associated with a value
Traceback:

File "/app/src/pages/model_results.py", line 1433, in <module>
    model_results()
    ~~~~~~~~~~~~~^^
File "/app/src/pages/model_results.py", line 865, in model_results
    if not gage_flow_ts.empty:
           ^^^^^^^^^^^^
```